tools/ansible/tasks/wsu/main.yaml

```diff
@@ -82,6 +217,7 @@
       debug:
         msg: "Windows temporary directory: {{ win_temp_dir.path }}"
 
+    # Uses winRM to transfer files over. Copies entire content or fails
```

Comment:
Please call out that checksum is performed given that it is not documented clearly.